lib/main_containers.pm

```diff
@@ -73,9 +73,12 @@ sub load_tests_docker_common {
 
 sub load_container_tests {
     my $runtime = get_required_var('CONTAINER_RUNTIME');
+    my $flavor  = get_required_var('FLAVOR');
     boot_hdd_image;
+
     # Container Image tests
-    if (get_var('FLAVOR', '') =~ /Container-Image/) {
+    my @flavor_list = ('Container-Image', 'Container-Image-Updates', 'NET');
+    if (grep { /$flavor/ } @flavor_list) {
```

Comment:
sounds like : 

```
sub is_container_images {
 my $flavor = @_;
 return  grep { /$flavor/ }  ('Container-Image', 'Container-Image-Updates', 'NET');
}
```

Reply:
looks like major part of `load_container_tests` is under this condition . In case it is not one of this flavors we will just boot and do coredump test . Is this is what we want ? Is there case when it really happens ?

Reply:
I don't think so.

Reply:
For now, there is not a case where this happens.